### PR TITLE
request user geo before continuing string

### PR DESCRIPTION
**Pull request by albatrocity**

### CODE REVIEW

File: lib/actions/userActions/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const i18n               = require('../../../config/i18n')
+const ch                 = require('../../chatHelpers')
+const allActions         = require('../')
+
+const actions = {
+  // Used to request most recent geo location of user
+  'user:get_geo': (payload) => {
+    return ch.respondWith({
+      text: ``,
+      responseType: `userLocation`,
```
Comment (albatrocity) [inline lib/actions/userActions/index.js]: @roblafeve check for a `responseType` of `userLocation`, get the updated location of the user, and handle the rest like a normal response.

Comment (roblafeve) [inline lib/actions/userActions/index.js]: @albatrocity yeah, that makes sense. Would it be worth outlining all possible action types in a wiki? I'm also fine with just digging into the userActions directly. Just a thought ¯\_(ツ)_/¯

Comment (albatrocity) [inline lib/actions/userActions/index.js]: @roblafeve https://github.com/cremalab/string-api#user-response-types

Comment (albatrocity) [inline lib/actions/userActions/index.js]: or rather: https://github.com/cremalab/string-api/tree/request-geo#user-response-types

Comment (roblafeve) [inline lib/actions/userActions/index.js]: Hadn't seen this. Thanks!

**Pull request merged**